lms/templates/help_modal.html

```diff
@@ -99,15 +99,19 @@
           <label data-field="email" for="feedback_form_email">${_('E-mail')}*</label>
           <input name="email" type="text" id="feedback_form_email" required>
     % endif
+          <div class="js_feedback_form_course_select hidden">
```

Comment:
Is `hidden` the correct class to use to keep this hidden from users, or is there something else I should use (the rest of this template seems to set the `display` attribute directly).